app/wama.web/wwwroot/css/site.css

```diff
@@ -8,6 +8,9 @@ input#check-in-user-id {
     text-align: center;
 }
 
+.large{
```

Comment:
Okay. Would you like the numbers to be more visible (larger) in some sort or on the same line as the indication. eg 
Employees check in: 35